### PR TITLE
DP-23272 Improvements to Guide header (clean)

### PR DESCRIPTION
**Pull request by clairesunstudio**
https://github.com/massgov/mayflower/pull/1552

Any PRs being created needs a changelog.txt file before being merged into dev. See: [Change Log Instructions](https://github.com/massgov/mayflower/blob/develop/docs/for-developers/changelog-instructions.md)

## Description

- When there isn't an image, a class is added to illustrated header container and not just the image in order to adjust the content width.
- Adjusted Illustrated header width to match the new comps. (see ticket)

## Related Issue / Ticket

- [JIRA issue](https://massgov.atlassian.net/browse/DP-23272)
- [Github issue]()

## Steps to Test

1. There is a new PL element to show this case, open "organisms" > "by template" > "Illustrated Header no image" in PatternLab and validate against the mockup.

## Screenshots

j-r-capture 2021-11-12 a la(s) 10 50 28

## Additional Notes:

Anything else to add?

#### Impacted Areas in Application

*

#### @TODO

*

### CODE REVIEW

File: packages/assets/scss/03-organisms/_illustrated-header.scss [hunk 1]
```diff
@@ -105,6 +98,30 @@ $bp-illustrated-header-image-width: 360px;
...
   }
+
+  &--no-image {
+    .ma__illustrated-header__image {
+      display: none;
+    }
+
+    .ma__illustrated-header__content {
+      @media ($bp-page-width-min) {
+        width: 935px;
```
Comment (clairesunstudio) [inline packages/assets/scss/03-organisms/_illustrated-header.scss:109]:
```suggestion
        width: 935px;

```
How is this width determined? Are we using this title width in other headers?

**Comment by clairesunstudio** [comment]
I consolidated the docs into one file. Currently the no image variant is using the same check as the colored variant, and is breaking the that variant. https://mayflower.digital.mass.gov/patternlab/b/patternlab/DP-23272_guide-header-clean/?p=organisms-illustrated-header-no-image
Prod: https://mayflower.digital.mass.gov/patternlab/?p=organisms-illustrated-header-colored
We should come up with a better name to differentiate them and a conditional logic for both variants to exist. 
Please also add a changelog to this PR.

File: packages/assets/scss/03-organisms/_illustrated-header.scss (hunk 1 above)
Comment (Javi-er) [reply packages/assets/scss/03-organisms/_illustrated-header.scss:109]: The width comes from the mockup, the new topic page header is somewhat similar: https://mayflower.digital.mass.gov/patternlab/b/patternlab/DP-22640_topic-page-redesign-fe/?p=organisms-page-banner-as-white
Although in that case is 1050px width.

File: changelogs/DP-23272.yml [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+#    description: Describe the change and add the PR number. see ___DESCRIPTION___ e.g. Adds apples to apple trees for admin apple pickers (#PR)
+#    issue: Add a Jira ticket or issue number, e.g. DP-12345 or 124
+#    impact: Enter a valid impact, e.g. Minor
+
+Changed:
+  - project: PatternLab
+    component: IllustratedHeader
+    description: Adding a new white variant. (#1561)
+    issue: DP-23272
+    impact: Minnor
```
Comment (clairesunstudio) [inline changelogs/DP-23272.yml]:
```suggestion
    impact: Minor
```

File: changelogs/DP-23272.yml [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+#  - project: Enter one or multiple (comma separated) valid project prefix(es) - see ___PROJECT___ e.g. React, Patternlab
+#    component: Enter one or multiple (comma separated) valid component name(s) - see ___COMPONENT___ e.g. Color
+#    description: Describe the change and add the PR number. see ___DESCRIPTION___ e.g. Adds apples to apple trees for admin apple pickers (#PR)
+#    issue: Add a Jira ticket or issue number, e.g. DP-12345 or 124
+#    impact: Enter a valid impact, e.g. Minor
+
+Changed:
+  - project: PatternLab
+    component: IllustratedHeader
+    description: Adding a new white variant. (#1561)
```
Comment (clairesunstudio) [inline changelogs/DP-23272.yml]:
```suggestion
    description: Add a no-background variant. (#1561)
```

File: packages/patternlab/styleguide/source/_patterns/03-organisms/by-template/illustrated-header.md [hunk 4]
```diff
@@ -8,9 +8,10 @@ This is displays a Page Header pattern with an optional image off to the right
 * Page Header
 
 ### Variant options
-* If the image isn't used a [green](./?p=organisms-illustrated-header-colored) background is shown in it's place.
-* There is an inverted variation which uses a [blue](./?p=organisms-illustrated-header-inverted) background in place of the white in the standard variant.
-* The image-centered variant is used for images with a [custom focal point] (./?p=organisms-illustrated-header-image-centered)
+* Colored: If the image isn't used a [green](./?p=organisms-illustrated-header-colored) background is shown in it's place.
+* Inverted: There is an inverted variation which uses a [blue](./?p=organisms-illustrated-header-inverted) background in place of the white in the standard variant.
+* Image Centered: The image-centered variant is used for images with a [custom focal point] (./?p=organisms-illustrated-header-image-centered). Set `bgCentered` to true. 
+* Plain: A plain header with on text and no image or color background.
```
Comment (clairesunstudio) [inline packages/patternlab/styleguide/source/_patterns/03-organisms/by-template/illustrated-header.md]:
```suggestion
* No-Background: A variant with no image or color background. Set `noImage` to true

```

File: packages/assets/scss/03-organisms/_illustrated-header.scss (hunk 1 above)
Comment (clairesunstudio) [reply packages/assets/scss/03-organisms/_illustrated-header.scss:109]: ok, thanks for noting that. I will approve this for now, but for the topic header, let's reconvene with Marissa, I think should come up something more consistent for page headers across the board. 

For references, the width on desktop for the page content is 738px, and we have these numbers in the system:

$l-max-content-width:           820px;
$l-max-content-wrapper:         1240px;
// Semi Global breakpoint media variables
// switch the header from full width to hamburger
$bp-header-toggle-min:      "min-width: 841px" !default;
$bp-header-toggle-max:      "max-width: 840px" !default;
$bp-xs:   480px !default;
$bp-s:    620px !default;
$bp-m:    780px !default;
$bp-l:    910px !default;
$bp-l-ex: 1050px !default;
$bp-xl:   1200px !default;

**Pull request merged**